AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockInsertRenditionImageFromGalleryFileSizeTest.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminAdobeStockInsertRenditionImageFromGalleryFileSizeTest">
+        <annotations>
+            <features value="AdminAdobeStockInsertRenditionImageFromGalleryFileSizeTest"/>
+            <useCaseId value="https://github.com/magento/adobe-stock-integration/issues/1806"/>
+            <title value="Admin user should see correct image file size after rendition inserted from gallery"/>
+            <testCaseId value="https://studio.cucumber.io/projects/131313/test-plan/folders/1507933/scenarios/5200023"/>
+            <stories value="User inserts image rendition to the content from media gallery"/>
+            <description value="Admin user should see correct image file size after rendition inserted from gallery"/>
+            <severity value="AVERAGE"/>
+            <group value="adobe_stock_media_gallery"/>
+        </annotations>
+        <before>
+            <createData entity="SimpleSubCategory" stepKey="category"/>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsAdmin"/>
+            <!-- Prepare configuration -->
+            <magentoCLI command="config:set system/media_gallery_renditions/width 50" stepKey="prepareWidth"/>
```

Comment:
Please introduce an action group to set configuration values to MediaGalleryRenditions modules (please create a new pull request to magento2 for this purpose)